rollbar/lib/transforms/__init__.py

```diff
@@ -1,6 +1,21 @@
+try:
+    # Python 3
+    from collections.abc import Iterable
+except ImportError:
+    # Python 2.7
+    from collections import Iterable
+
 from rollbar.lib import (
-    python_major_version, binary_type, string_types, integer_types,
-    number_types, traverse)
+    python_major_version,
+    binary_type,
+    string_types,
+    integer_types,
+    number_types,
+    traverse,
+    type_info,
+)
+from rollbar.lib.transform import Transform
```

Comment:
With the `Transform` class being moved to another package if we ever removed this import, it would break backwards compatibility for anyone with a custom transform trying to import `Transform` from `rollbar.lib.transforms`. This seems a little fragile and may warrant a comment explaining the need for it to stay until a major release.